src/commands/run-tests.yml

```diff
@@ -1,12 +1,17 @@
 description: "Test with RSpec."
+parameters:
+  additional-tests:
+    description: The steps to run for the additional rspec
+    type: steps
+    default: []
 steps:
   - run:
       name: Run Specs
       command: |
         mkdir /tmp/test-results
         TESTFILES=$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
         bundle exec rspec --colour --require fivemat --format Fivemat --order rand --format RspecJunitFormatter --out test-results/rspec.xml -- ${TESTFILES}
-
+  - steps: << parameters.additional-tests >>
```

Comment:
I was trying to find documentation for nested `steps` but couldn't find it. Have you verified this works?

Reply:
Its available in orb only- https://circleci.com/docs/2.0/reusing-config/#parameter-types

Reply:
Perfect, thanks.